config/rating.yaml

```
@@ -14,5 +14,8 @@ ingress:
       hosts:
         - rating.jenkins.io
 
+nodeSelector:
+  agentpool: arm64small
```

```suggestion
  agentpool: x86medium
```

We haven't evaluated if an ARM image is available for this app